File: helloworld.c
#include <stdio.h> 
  
int main() 
{ 
  
    // Declare the variables
    char str[20]; 
  
    // --- Integer --- 
  
    // Input the integer 
    printf("Enter with your name: "); 
    gets(str); 
    printf("Hello World - %s", str);
    return 0;
}

File: HelloWorld.java
import java.util.Scanner;

public class HelloWorld {
    public static void main(String[] args) {
        String name = "";
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter your name: ");
        name = sc.next();
        System.out.println(String.format("Hello World - %s!", name));
    }
}